onnxscript/rewriter/ort_fusions/gelu.py

```diff
@@ -33,5 +33,10 @@ def rewrite(self, op, x):
 gelu_rules = pattern.RewriteRuleSet([_rule])
 
 
-def fuse_gelu(model: ir.Model) -> None:
-    gelu_rules.apply_to_model(model)
+def fuse_gelu(model: ir.Model, debug: bool = False) -> int:
+    count = gelu_rules.apply_to_model(model)
```

Comment:
Reminds me: may be useful to lift this pattern (which is repeated in every fuse_* method) into a utility.

Reply:
Add utility to _fusion_utils.py. Let me if you think there is a better place for this.